From 573cea67cc675c9a6dbfe5c6296047ea6964afac Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Sat, 14 Mar 2026 11:43:35 +0100
Subject: [PATCH 3/8] feat: enable custom plugins

---
 init.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index b56ce29..e6a3c32 100644
--- a/init.lua
+++ b/init.lua
@@ -944,7 +944,7 @@ require('lazy').setup({
   --    This is the easiest way to modularize your config.
   --
   --  Uncomment the following line and add your plugins to `lua/custom/plugins/*.lua` to get going.
-  -- { import = 'custom.plugins' },
+  { import = 'custom.plugins' },
   --
   -- For additional information with loading, sourcing and examples see `:help lazy.nvim-🔌-plugin-spec`
   -- Or use telescope!
-- 
2.53.0

