From 6bdd22aeadeca5495aeb4ef158652e7c72e6f4c7 Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Sat, 14 Mar 2026 15:52:39 +0100
Subject: [PATCH 5/8] feat: enable nerd font

---
 init.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index bd1964e..37be514 100644
--- a/init.lua
+++ b/init.lua
@@ -91,7 +91,7 @@ vim.g.mapleader = ' '
 vim.g.maplocalleader = ' '
 
 -- Set to true if you have a Nerd Font installed and selected in the terminal
-vim.g.have_nerd_font = false
+vim.g.have_nerd_font = true
 
 -- [[ Setting options ]]
 -- See `:help vim.o`
-- 
2.53.0

