From 4294ec407a6be2e44e9c82a8685fc0f21331957a Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Sat, 14 Mar 2026 15:52:39 +0100
Subject: [PATCH 6/9] feat: enable nerd font

---
 init.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index ac3e913..54fdf1a 100644
--- a/init.lua
+++ b/init.lua
@@ -91,7 +91,7 @@ vim.g.mapleader = ' '
 vim.g.maplocalleader = ' '
 
 -- Set to true if you have a Nerd Font installed and selected in the terminal
-vim.g.have_nerd_font = false
+vim.g.have_nerd_font = true
 
 -- [[ Setting options ]]
 -- See `:help vim.o`
-- 
2.53.0

