From 3cc1904472b983ce4d5ad5034ef565828d30b0f3 Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Sat, 14 Mar 2026 16:01:35 +0100
Subject: [PATCH 7/7] feat: info about applying patches

---
 apply.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apply.sh b/apply.sh
index 493d760..befb2c8 100755
--- a/apply.sh
+++ b/apply.sh
@@ -83,4 +83,5 @@ git for-each-ref \
     || true
 echo ""
 success "Done. You are now on branch '$(git symbolic-ref --short HEAD)'."
+info "Applying patches from 'patches/' directory"
 git am patches/*.patch
-- 
2.53.0

