From 0cbc886f16ecefc6552fed05cd227400f1a99c6a Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Sun, 15 Mar 2026 08:49:29 +0100
Subject: [PATCH 7/7] feat: enable couple of ls-es

---
 init.lua | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init.lua b/init.lua
index 54fdf1a..cbc4c20 100644
--- a/init.lua
+++ b/init.lua
@@ -600,9 +600,9 @@ require('lazy').setup({
       --  See `:help lsp-config` for information about keys and how to configure
       ---@type table<string, vim.lsp.Config>
       local servers = {
-        -- clangd = {},
+        clangd = {},
         -- gopls = {},
-        -- pyright = {},
+        pyright = {},
         -- rust_analyzer = {},
         --
         -- Some languages (like typescript) have entire language plugins that can be useful:
-- 
2.53.0

