From 055457f47ca73c5646350aa4fdfb47cd85cca31c Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Thu, 19 Mar 2026 15:32:40 +0100
Subject: [PATCH 8/8] feat: change preset for luasnip accept

---
 init.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index 07c96a3..ecf078a 100644
--- a/init.lua
+++ b/init.lua
@@ -760,7 +760,7 @@ require('lazy').setup({
         -- <c-k>: Toggle signature help
         --
         -- See :h blink-cmp-config-keymap for defining your own keymap
-        preset = 'default',
+        preset = 'enter',
 
         -- For more advanced Luasnip keymaps (e.g. selecting choice nodes, expansion) see:
         --    https://github.com/L3MON4D3/LuaSnip?tab=readme-ov-file#keymaps
-- 
2.53.0

