From 84fea3d80d63e661d9e192e6f9805b60f6a178f5 Mon Sep 17 00:00:00 2001
From: OwnSample <ownsample@tutanota.com>
Date: Sun, 15 Mar 2026 09:04:59 +0100
Subject: [PATCH 8/9] feat: enable c cpp format on save

---
 init.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index cbc4c20..2aaa981 100644
--- a/init.lua
+++ b/init.lua
@@ -684,7 +684,7 @@ require('lazy').setup({
         -- Disable "format_on_save lsp_fallback" for languages that don't
         -- have a well standardized coding style. You can add additional
         -- languages here or re-enable it for the disabled ones.
-        local disable_filetypes = { c = true, cpp = true }
+        local disable_filetypes = {}
         if disable_filetypes[vim.bo[bufnr].filetype] then
           return nil
         else
-- 
2.53.0

